test/unit/models/airport.js

```diff
@@ -0,0 +1,38 @@
+import { assert } from 'chai';
+
+import models from 'server/models';
+
+import factories from 'test/factories';
+import truncate from 'test/truncate';
+
+describe('Airport model', () => {
+  let airport;
+
+  beforeEach(async () => {
+    await truncate();
+
+    airport = await factories.airport();
+  });
+
+  it('should create Airport ID as number', async () => {
+    assert.isNumber(airport.id);
+  });
+
+  it('should create Airport Name as String', async () => {
+    assert.isString(airport.name);
+    for(var i =0; i < airport.name.length;i++) assert.isNotNumber(airport.name.charAt(i)); 
+  });
+
+  it('should create Airport code as 3 char string', async () => {
+    assert.isString(airport.code);
+    assert.lengthOf(airport.code, 3);
```

Comment:
Ok, this one here defeats the entire purpose of testing, this is called `false negative` testing.

It's like testing `true === true` considering that in the factory you always pass a string with the length of 3.

What needs to be done is:

1. Update the model to allow maxLength of N(3 in our case)
2. Write a test that tries to create an airport with a code length of 2 and then 4 and check that it fails/throws an error.

Same with all the other stuff. If you pass exactly what you expect, the tests are not really usefull 🙈